js/piwik.js

```diff
@@ -3901,7 +3927,7 @@ if (typeof window.Piwik !== 'object') {
              */
             function getClassesRegExp(configClasses, defaultClass) {
                 var i,
-                    classesRegExp = '(^| )(piwik[_-]' + defaultClass;
+                    classesRegExp = '(^| )(piwik[_-]' + defaultClass + '|matomo[_-]' + defaultClass;
```

Comment:
will need to document the support of these new attributes eg `matomo_download`. We won't remove support for the `piwik` class names any time soon I suppose